shared/AppInsightsCommon/src/TelemetryItemCreator.ts

```diff
@@ -32,21 +32,21 @@ export class TelemetryItemCreator {
                 throw Error("Input doesn't contain all required fields");
         }
 
-        let telemetryItem: ITelemetryItem = {
+        const telemetryItem: ITelemetryItem = {
             name: envelopeName,
             time: new Date().toISOString(),
             iKey: "", // this will be set in TelemetryContext
             ext: systemProperties ? systemProperties : {}, // part A
             tags: [],
             data: {
             },
-            baseType: baseType,
+            baseType,
```

Comment:
Is this actually taking the parameter value?

Reply:
yes, this is following [object-literal-shorthand rule](https://palantir.github.io/tslint/rules/object-literal-shorthand/) that enforce using [es6 object literal shorthand](https://ariya.io/2013/02/es6-and-object-literal-property-value-shorthand).